packages/function-entry/controller/src/index.js

```diff
@@ -0,0 +1,79 @@
+import debug from 'debug';
+
+const log = debug('@pie-element:function-entry:controller');
+
+const process = (v, ignoreWhitespace) => {
+    let out = v ? v.trim() : '';
+    out = !ignoreWhitespace ? out : out.replace(/ /g, '');
+    return out;
+};
+
+const isResponseCorrect = (correctResponse, model, value) => {
+    const processedValue = process(value, model.ignoreSpacing);
```

Comment:
We'll need to do more than just a string compare here, we were talking about using `mathjs.simplify`?